libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Recognizers/EntityRecognizers/EntityRecognizer.cs

```diff
@@ -1,27 +1,122 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Bot.Schema;
 using Microsoft.IdentityModel.Logging;
 using Microsoft.Recognizers.Text;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Recognizers
 {
     /// <summary>
     /// Entity recognizers base class.
     /// </summary>
-    public class EntityRecognizer
+    public class EntityRecognizer : Recognizer
```

Comment:
Isn't this a non-backward compatible breaking change?

Reply:
no, it's not breaking change.